src/summersplash2022/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { NextPage, InferGetStaticPropsType } from 'next';
+import DefaultErrorPage from 'next/error';
+import { getStaticProps } from 'pages';
+import Content from './Content';
+import { rqdp } from './utils/utils';
+
+const Summersplash2022: NextPage<
+  InferGetStaticPropsType<typeof getStaticProps>
+> = () => {
+  const router = useRouter();
+
+  const queryedPage = rqdp(router);
+
+  const [mode, setMode] = useState(
+    rqdp(router) === 'sommerjobb' ? 'internship' : 'job',
+  );
+
+  useEffect(() => {
+    setMode(rqdp(router) === 'sommerjobb' ? 'internship' : 'job');
+  }, [rqdp(router)]);
```

Comment:
Samme som over, bare "viktigere". useEffects og slike ting forventer noe statisk analyserbar kode som "dependencies" som da sendes inn som andre argument. Så det er alltid fint å sende spesifikke variabler.

```suggestion
  }, [queryedPage]);
```